youtube_dl/extractor/rai.py

```diff
@@ -103,22 +103,33 @@ def _extract_relinker_info(self, relinker_url, video_id):
         }.items() if v is not None)
 
     @staticmethod
-    def _extract_subtitles(url, subtitle_url):
-        subtitles = {}
-        if subtitle_url and isinstance(subtitle_url, compat_str):
-            subtitle_url = urljoin(url, subtitle_url)
+    def _extract_subtitles(url, video_data):
+
+        def create_sub(url, lang):
             STL_EXT = '.stl'
             SRT_EXT = '.srt'
-            subtitles['it'] = [{
-                'ext': 'stl',
-                'url': subtitle_url,
+            sub = {}
+            sub[lang] = [{
+                'ext': determine_ext(url),
+                'url': url,
             }]
-            if subtitle_url.endswith(STL_EXT):
-                srt_url = subtitle_url[:-len(STL_EXT)] + SRT_EXT
-                subtitles['it'].append({
+            if url.endswith(STL_EXT):
+                srt_url = url[:-len(STL_EXT)] + SRT_EXT
+                sub[lang].append({
                     'ext': 'srt',
                     'url': srt_url,
                 })
+            return sub
+
+        subtitles = {}
+        subtitlesArray = video_data.get('subtitlesArray')
```

Comment:
use the common naming convention in Python.

Reply:
fixed